relisten/player/relisten_player_queue.tsx

```diff
@@ -44,15 +46,25 @@ function nextQueueTrackId() {
 
 export class PlayerQueueTrack {
   public readonly identifier: string;
+  public readonly title: string;
+  public readonly artist: string;
+  public readonly albumTitle: string;
```

Comment:
yeah this was exactly what i was thinking, except the callsites should pass it in this object directly instead of only providing the `SourceTrack`